cfgov/unprocessed/js/modules/util/Behavior.js

```diff
@@ -0,0 +1,34 @@
+/* ==========================================================================
+   Get Breakpoint State
+   ========================================================================== */
+
+'use strict';
+
+var Behavior = {};
+
+/**
+ * @param {string} selector Used to query dom for elements.
+ * @param {string} event Event type to add to element.
+ * @param {function} eventHandler Callback for event.
+ */
+Behavior.create = function create( selector, event, eventHandler ) {
```

Comment:
Ah right, dom-events is passing a list of events onto one element, this is passing a selector for (potentially) a number of elements one event.